test/common/router/config_impl_test.cc

```diff
@@ -5256,13 +4921,13 @@ name: ZeroIdleTimeout
       - match: { regex: "/regex"}
         route:
           cluster: some-cluster
-          idle_timeout: 0s
+          idle_timeout: 1s
```

Comment:
Why did this change? Validation?

Reply:
Yes, [protobuf validation](https://github.com/envoyproxy/envoy/blob/master/api/envoy/api/v2/route/route.proto#L574). (I'm no protobuf expert, but appears `(validate.rules).duration.gt = {}` is equivalent to > 0?)

I'm actually going to change this back to zero, but use `EXPECT_THROW` instead as that's the new expected behavior in v2. (`ExplicitIdleTimeout` below already tests a good `idle_timeout` so this test would be redundant)